src/components/Pega_Extensions_CaseLauncher/styles.ts

```diff
@@ -0,0 +1,22 @@
+// utilizing theming, comment out, if want individual style
+import styled, { css } from 'styled-components';
+import { Configuration, defaultThemeProp } from '@pega/cosmos-react-core';
+
+export default styled(Configuration)``;
```

Comment:
move Configuration to the index.js - We avoid in this repo doing that in the styles.ts - see other components